exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_log_exporter.h

```diff
@@ -0,0 +1,89 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#ifdef ENABLE_LOGS_PREVIEW
+
+// clang-format off
+
+#  include "opentelemetry/exporters/otlp/protobuf_include_prefix.h"
+#  include "opentelemetry/proto/collector/logs/v1/logs_service.grpc.pb.h"
+#  include "opentelemetry/exporters/otlp/protobuf_include_suffix.h"
+
+// clang-format on
+
+#  include "opentelemetry/exporters/otlp/otlp_environment.h"
+#  include "opentelemetry/exporters/otlp/otlp_grpc_exporter_options.h"
+#  include "opentelemetry/exporters/otlp/otlp_recordable.h"
+#  include "opentelemetry/exporters/otlp/otlp_recordable_utils.h"
```

Comment:
Do we need all these headers - at least `otlp_recordable.h` and `otlp_recordable_utils.h` shouldn't be part of public API.

Reply:
Good catch, removed both headers.